optuna/cli.py

```diff
@@ -221,19 +213,21 @@ def _format_output(
         raise CLIUsageError(f"Optuna CLI does not supported the {output_format} format.")
 
 
-class _BaseCommand(Command):
-    def __init__(self, *args: Any, **kwargs: Any) -> None:
-
-        super().__init__(*args, **kwargs)
+class _BaseCommand:
+    def __init__(self) -> None:
         self.logger = optuna.logging.get_logger(__name__)
 
+    def add_arguments(self, parser: ArgumentParser) -> ArgumentParser:
```

Comment:
Could you add a docstring for each method of `_BaseCommand`?

Reply:
done.